xonsh/lib/subprocess.xsh

```diff
@@ -0,0 +1,19 @@
+"""Xonsh extension of the standard library subprocess module, using xonsh for
+subprocess calls"""
+from xonsh.imphooks import install_hook
+
+install_hook()
```

Comment:
Same here, you don't need the install hook, since you have it in the `__init__.py`